### ISSUE TITLE
Add ignoreProperties: [] option to color-named

### ISSUE THREAD

**Issue by christianvuerings** [stylelint/stylelint#1762]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

When you add a color name (which is a CSS class) in `composes`, it triggers a warning.

> Which rule, if any, is this issue related to?

`color-named`

> What CSS is needed to reproduce this issue?

```css
.Grid {
  composes: relative blue from "../index.css";
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "color-named": "never"
  }
}
```

> Which version of stylelint are you using?

`7.1.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

"CLI with `./node_modules/stylelint/dist/cli.js ./src/**/*.css`"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

"Yes, it's related to CSS modules."

> What did you expect to happen?

"No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

"The following warnings were flagged:"

```shell
src/Grid/Grid.css
Unexpected named color "blue"
```

Relates to #1724 #1712
Ideally, we would be able to have a similar fix by adding `composes` to the following list:
https://github.com/stylelint/stylelint/blob/9807f6de97625bacd8f9b3b74b300cb9eb1b459a/src/reference/propertySets.js
